### PR TITLE
Gravatar profile

### LINKED ISSUE #6813: Profile Image: If user has no profile image use Gravatar
Many users already have Gravatar. Please use a gravatar if the user has no profile image to display on attendee lists and other places.

### PR DESCRIPTION
**Pull request by tanay123456789**
Fixes #6813 

#### Changes proposed in this pull request:
Added ember addon for gravatar image

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/models/attendee.js [hunk 1]
```diff
@@ -35,6 +35,7 @@ export default ModelBase.extend({
   linkedin             : attr('string'),
   gender               : attr('string'),
   ageGroup             : attr('string'),
+  profileImage         : attr('number'),
```
Comment (MeghalBisht) [inline app/models/attendee.js:38]: You cannot add fields here until you add this field in server as well

File: app/templates/admin.hbs [hunk 2]
```diff
@@ -17,7 +17,7 @@
         {{t 'Users'}}
       </LinkTo>
       <LinkTo @route="admin.video-channels" class="item">
-        {{t 'Video Channels'}}
```
Comment (MeghalBisht) [inline app/templates/admin.hbs:20]: Revert

File: app/templates/components/forms/orders/attendee-list.hbs [hunk 3]
```diff
@@ -62,6 +62,16 @@
           </h4>
           <div class="ui two column divided relaxed stackable grid ">
             <div class="row">
+              <div class="column">
+                <h4 class="weight-300">
+                  {{t 'Profile Image'}}
+                </h4>
+                {{#if this.data.profileImage==undefined}}
```
Comment (MeghalBisht) [inline app/templates/components/forms/orders/attendee-list.hbs:69]: This is not how things work in ember. Checkout `if helper`  in ember

File: app/templates/components/forms/orders/attendee-list.hbs [hunk 4]
```diff
@@ -62,6 +62,16 @@
...
+                {{gravatar-image email=this.data.email}}
```
Comment (MeghalBisht) [inline app/templates/components/forms/orders/attendee-list.hbs:70]: This is not how things work in ember. Checkout helpers in ember

**Comment by MeghalBisht** [comment]
Please also do paste a working screenshot of the changes done as then you will yourself realise that changes you made will throw an error immediately when you run the app. All tests have failed because syntax is wrong

**Comment by tanay123456789** [comment]
@MeghalBisht ok doing the required changes

**Comment by MeghalBisht** [comment]
@tanay123456789 update?

**Comment by maze-runnar** [comment]
@tanay123456789 I have told you https://github.com/fossasia/open-event-frontend/pull/7073 that syntax is wrong. And closed that. Now you have created PR with same syntax again. And please run project locally, check if it actually fixing something, or atleast not breaking anything. Then if you stuck after solving even 10% of issue then ask. But run it once locally, and read about syntax.

https://www.npmjs.com/package/ember-truth-helpers

**Comment by tanay123456789** [comment]
@maze-runnar  yes sir now i know that where I did wrong.  I am doing the required  changes

**Comment by tanay123456789** [comment]
@MeghalBisht @maze-runnar   right now I know that where i was wrong and which files are needed to change. I have a basic idea of what to change but I am struggling with implementation and I am trying my level best.If possible guide me or you can give it to others if it is very important.

**Comment by MeghalBisht** [comment]
> If possible guide me or you can give it to others if it is very important.

You can join the weekly meetings on Tues & Thursday

File: app/models/attendee.js (hunk 1 above)
Comment (tanay123456789) [reply app/models/attendee.js:38]: @MeghalBisht 

 
> You can join the weekly meetings on Tues & Thursday
Where it is hosted?

**Comment by tanay123456789** [comment]
@MeghalBisht 
>> You can join the weekly meetings on Tues & Thursday
where it is hosted?

**Comment by klsingh016** [comment]
can i work?

**Comment by sachinchauhan2889** [comment]
hello @tanay123456789 , sorry i am closing this PR since this can easily be handle in server repo.
We are getting gravatar url from server thats why it is server related issue. I checked this today while reviewing your PR.
Server is quite complex, hence i have created a PR there. https://github.com/fossasia/open-event-server/pull/7969
Please take other issue.

**Comment by tanay123456789** [comment]
@sachinchauhan2889  ok sir

**Pull request closed without merging**